content/guides/models/registry/configure_registry.md

```diff
@@ -7,24 +7,55 @@ title: Configure registry access
 weight: 3
 ---
 
-<!-- A registry, and the linked artifacts inside a registry, belong to an organization. This means that teams within an organization can publish and consume artifacts linked to a registry, if that team has correct access control. -->
-
-Registry admins can limit who can access a registry by navigating to a registry's settings and assigning a user's role to [Admin, Member, or Viewer]({{< relref "#registry-roles-permissions" >}}). Users can have different roles in different registries. For example, a user can have a view role in "Registry A" and a member role in the "Registry B". 
+Add, remove, and manage registry roles for teams and individual users in a registry by configuring a registry's settings.
 
 {{% alert %}}
-Only registry admins can [restrict visibility]({{< relref "#restrict-visibility-to-a-registry" >}}), [configure user roles]({{< relref "#configure-user-roles-in-a-registry" >}}), or [remove users]({{< relref "#remove-a-user-from-a-registry" >}}) from registries in an organization.
+Only team or registry administrators can [restrict visibility]({{< relref "configure_registry.md#restrict-visibility-to-a-registry" >}}), [configure registry roles]({{< relref "configure_registry.md#configure-registry-roles" >}}), [add users]({{< relref "configure_registry.md#add-a-user-from-a-registry" >}}), or [remove users]({{< relref "configure_registry.md#remove-a-user-from-a-registry" >}}) from a registry.
 {{% /alert %}}
 
-## Registry roles permissions
 
-A user within an organization can have different roles, and therefore permissions, for each registry in their organization.
+## Add a user or a team to a registry
+
+Team or registry administrators can add individual users or entire teams to a registry. To add a user or team to a registry:
+
+1. Navigate to the Registry App at https://wandb.ai/registry/.
+2. Select the registry you want to add a user or team to.
+3. Click on the gear icon on the upper right hand corner to access the registry settings.
+4. Navigate to the **Registry access** section.
+5. Click on the **Add access** button.
+6. Add a user name, email, or the name of a team to the **Include users and teams** field.
+7. Click **Add access**.
+
 
-{{% alert title="W&B role types" %}}
-W&B has three different types of roles: Organization roles, [team roles]({{< relref "/guides/models/app/settings-page/teams.md#team-roles-and-permissions" >}}), and [registry roles]({{< relref "#registry-roles-permissions" >}}).
 
-Your role in a team has no impact or relationship on your role in any registry.
+
+W&B attempts to assign users added to a registry a `Viewer` registry role by default. If the user has a [team role]({{< relref "/guides/models/app/settings-page/teams.md#team-role-and-permissions" >}}) in the organization, W&B assigns the user the highest level of privilege as their registry role. See [Resolve role conflicts]({{< relref "configure_registry.md#resolve-role-conflicts" >}}) for more information.
```

Comment:
Registry roles actually have nothing to do with the user's role in a team. A user can get a role in a registry if one of their teams is assigned a role. 

If a user already has a role in the registry, and a team that they are a member of gets assigned another role, their permission set includes the permissions given by both roles. 

Feel free to re-word this, I just wanted to explain how this works!

Reply:
Thanks! Re worded in next commit :)